plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareManager.java

```diff
@@ -42,6 +44,9 @@
     static final ConfigKey<String> s_vmwareSearchExcludeFolder = new ConfigKey<String>("Advanced", String.class, "vmware.search.exclude.folders", null,
             "Comma seperated list of Datastore Folders to exclude from VMWare search", true, ConfigKey.Scope.Global);
 
+    static final ConfigKey<Integer> s_vmwareOVAPackageTimeout = new ConfigKey<Integer>(Integer.class, "vmware.package.ova.timeout", "Advanced",String.valueOf(VM_OVA_PACKAGE_TIMEOUT_SEC),
```

Comment:
Fix a space after `,`. Remove the `VM_OVA_PACKAGE_TIMEOUT_SEC` and `SEC_TO_MILLIS_MULTIPLIER`.

Reply:
Will add the space after ,
Why to remove the constants  for default value and multiplier?

Reply:
@mrunalinikankariya see ` "Advanced",String.valueOf` would look better as ` "Advanced", String.valueOf` this is what I meant by fixing the space.
The constant are default values for the ConfigKey, and not used anywhere else. ConfigKey defaul values are themselves treated as well `defaults`.